pkg/knuu/knuu.go

```diff
@@ -69,6 +65,10 @@ func New(ctx context.Context, opts Options) (*Knuu, error) {
 		return nil, err
 	}
 
+	if err := k.loadEnvVariables(); err != nil {
```

Comment:
hmmm, this is a challenge I think.  Currently the `setDefaults` doesn't rely on any env vars but that could change in the future. 

I think it is ok to leave the `loadEnvVariables` at the top of the function and uses the default logrus logger for the one log line about the `.env` file so that we don't create a bigger issue in the future. 

We could simply add a comment explaining why we are leaving it the way we are so that we don't try and update in the future.

Reply:
Currently knuu options are set via its `New` func. The only thing is the default logger which reads the log level from env var. I think that loading `.env` file is more for user's usage. honestly I am not a big fan of it to be called with the `New` func. This is something that User had to take care of.

Reply:
I'll move to top anyways. created an issue to handle it #457